src/ssb.js

```diff
@@ -66,11 +66,58 @@ const createConnection = config => {
         }
         log("Initial connection attempt failed");
         log("Starting Scuttlebutt server");
+
+        const server = flotilla(ssbConfig);
         server(config);
+
+        const inProgress = {};
+        const maxHops = lodash.get(
+          config,
+          "friends.hops",
+          lodash.get(ssbConfig, "friends.hops", 0)
+        );
+
+        const add = address => {
```

Comment:
It looks to me like ssb-conn's (ssb-conn-hub's) connect function already handles this (connect() returns false if there's already a connection open or in progress), so `inProgress` and `add` probably aren't necessary.

Reply:
Nice catch! I noticed that the same person was showing up multiple times in my peer list, but maybe one was just `{ state: 'connecting' }`.